protocol-units/bridge/chains/ethereum/src/types.rs

```diff
@@ -0,0 +1,258 @@
+use std::collections::HashMap;
+
+use alloy::json_abi::Param;
+use alloy::network::{Ethereum, EthereumWallet};
+use alloy::primitives::{Address, FixedBytes};
+use alloy::providers::fillers::{
+	ChainIdFiller, FillProvider, GasFiller, JoinFill, NonceFiller, WalletFiller,
+};
+use alloy::providers::RootProvider;
+use alloy::pubsub::PubSubFrontend;
+use alloy::rlp::{RlpDecodable, RlpEncodable};
+use alloy::sol_types::SolEvent;
+use alloy::transports::BoxTransport;
+use bridge_shared::types::{
+	Amount, BridgeAddressType, BridgeHashType, BridgeTransferDetails, BridgeTransferId,
+	GenUniqueHash, HashLock, HashLockPreImage, InitiatorAddress, LockDetails, RecipientAddress,
```

Comment:
```suggestion
	HashLock, HashLockPreImage, InitiatorAddress, LockDetails, RecipientAddress,
```

Reply:
Won't continue as there are quite a few warnings for imports etc that would be good to clean up now

Reply:
Doing so now.